src/export/FSHExporter.ts

```diff
@@ -150,32 +154,46 @@ export class FSHExporter {
     return new Map().set('resources.fsh', results);
   }
 
-  private groupAsFilePerDefinition(): Map<string, Exportable[]> {
+  private groupAsFilePerDefinition(byGroup = false): Map<string, Exportable[]> {
     const files: Map<string, Exportable[]> = new Map();
     // Aliases, still get grouped into one file
     files.set('aliases.fsh', this.fshPackage.aliases);
 
     // Other definitions are each placed in an individual file
     for (const invariant of this.fshPackage.invariants) {
-      files.set(`${invariant.name}-Invariant.fsh`, [invariant]);
+      const fileDir = byGroup ? 'invariants' : '';
+      const filename = path.join(fileDir, `${invariant.name}-Invariant.fsh`);
```

Comment:
I don't think we still need to put the `-Invariant` after the name of the entity (this applies to all other types as well). Since the entities are in a folder which gives their type, for me that is sufficient, and having the name appended ends up being a bit redundant.

Reply:
That makes sense! I've removed them alL!